chore(grid):link editing/overview notes section (#378)
telerik/blazor-docs: components/grid/templates/editor.md

The column's `EditTemplate` defines the inline template or component that will be rendered when the user is [editing]({%slug components/grid/overview%}#editing) the field. It is also used when inserting a new item.

> You can data bind components in it to the current context, which is an instance of the model the grid is bound to. You will need a global variable that is also an instance of the model to store those changes. The model the template receives is a copy of the original model, so that changes can be cancelled with the `Cancel` command (more details on how and when that copy is created you can find in the notes section of [editing/overview](../editing/overview#notes) article).

Reviewer: ```suggestion
You can data bind components in the editor template to the current context, which is an instance of the model the grid is bound to. You will need a global variable that is also an instance of the model to store those changes. The template receives a copy of the original model, so that changes can be cancelled with the `Cancel` command. See the **Notes** section of the [Editing Overview]({%slug components/grid/editing/overview%}#notes) article for more details on how and when that copy is created.
```
